lib/geoblacklight/reference.rb

```diff
@@ -38,7 +38,13 @@ def to_hash
     # The URI used for this instance's creation
     # @return [String]
     def uri
-      @reference[0]
+      @reference[0] == 'http://www.isotc211.org/schemas/2005/gmd/' ? iso19139_parse(@reference[0]) : @reference[0]
```

Comment:
Could this be generalized to just remove trailing slashes from anything?

Reply:
yes I can update it

Reply:
I agree, we can just generalize this. Otherwise, PR looks good.

Reply:
👍 same.. maybe just even a regex? `'http://example.com///'.sub(/\/*$/, '')`

Reply:
@mejackreed  now i see your comment; yes I can update to remove any trailing slashes not just one and use a regex.